control-plane-operator/controllers/hostedcontrolplane/kas/params.go

```diff
@@ -77,9 +77,10 @@ type KubeAPIServerServiceParams struct {
 }
 
 const (
-	KonnectivityHealthPort      = 2041
-	KonnectivityServerLocalPort = 8090
-	KonnectivityServerPort      = 8091
+	KonnectivityHealthPort         = 2041
+	KonnectivityServerLocalPort    = 8090
+	KonnectivityServerPort         = 8091
+	KonnectivityServerCipherSuites = "TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305_SHA256,TLS_AES_128_GCM_SHA256,TLS_CHACHA20_POLY1305_SHA256,TLS_AES_256_GCM_SHA384"
```

Comment:
Where does this cipher list come from?

Reply:
These were considered secure according to https://ciphersuite.info/cs/?sort=sec-desc. But the list is not complete, so removed this and refactored the Konnectivity cipher handling a bit.

Reply:
Ultimately: there are security standards that the offering is measured against and anything with weak ciphers is not allowed. An example of this can be checked with namp against the konnectivity node port (note the warning in the output)

```
Host is up (0.016s latency).
rDNS record for 52.117.136.234: ea.88.7534.ip4.static.sl-reverse.com

PORT      STATE SERVICE
31069/tcp open  unknown
| ssl-enum-ciphers: 
|   TLSv1.2: 
|     ciphers: 
|       TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA (secp256r1) - C
|       TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA (secp256r1) - A
|       TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256 (secp256r1) - A
|       TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA (secp256r1) - A
|       TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384 (secp256r1) - A
|       TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256 (secp256r1) - A
|       TLS_RSA_WITH_3DES_EDE_CBC_SHA (rsa 2048) - C
|       TLS_RSA_WITH_AES_128_CBC_SHA (rsa 2048) - A
|       TLS_RSA_WITH_AES_128_GCM_SHA256 (rsa 2048) - A
|       TLS_RSA_WITH_AES_256_CBC_SHA (rsa 2048) - A
|       TLS_RSA_WITH_AES_256_GCM_SHA384 (rsa 2048) - A
|     compressors: 
|       NULL
|     cipher preference: client
|     warnings: 
|       64-bit block cipher 3DES vulnerable to SWEET32 attack
|   TLSv1.3: 
|     ciphers: 
|       TLS_AKE_WITH_AES_128_GCM_SHA256 (ecdh_x25519) - A
|       TLS_AKE_WITH_AES_256_GCM_SHA384 (ecdh_x25519) - A
|       TLS_AKE_WITH_CHACHA20_POLY1305_SHA256 (ecdh_x25519) - A
|     cipher preference: server
|_  least strength: C
````

Reply:
Tangibly what this cipher suite translates to at this time is only allowing TLS1.2 and 1.3 ciphers (and that over time as ciphers become insecure and new more secure ones are released) should change

Reply:
command to use: `nmap --script ssl-enum-ciphers`